redpanda/values.yaml

```diff
@@ -92,8 +92,15 @@ listeners:
     # The port for the admin server
     # Metrics are served off of this port at /metrics
     port: 9644
+    # Optional external section
     external:
-      enabled: false
+      # enabled: true
+      # Type of external access (options are NodePort and LoadBalancer)
+      # type: NodePort
+      # External port
+      # This listener port will be used for the external port if this is not included
+      port: 31644
```

Comment:
This section is optional for each listener. If it doesn't exist and external is enabled for all listeners, then the NodePort service will attempt to use the port (in this case on line 94) as the NodePort. By default this will result in the service failing to start with an error if you haven't configured Kubernetes to expand the NodePort range. I included details on how to do this for minikube in the startup section of the main README. I wasn't able to find details on how to do this in kind though, and this is entirely optional.